src/ecdsa/test_ecdh.py

```diff
@@ -259,22 +259,27 @@ def test_ecdh_der():
     assert sharedsecret == unhexlify(gshared_secret)
 
 
+# Exception classes used by run_openssl.
+class RunOpenSslError(Exception):
```

Comment:
what's wrong with subprocess.SubprocessError?

Reply:
because python 2.x have no error there
```>       except subprocess.SubprocessError:
E       AttributeError: 'module' object has no attribute 'SubprocessError'
```